studio-docs/source/org/members.mdx

```diff
@@ -1,19 +1,83 @@
 
-Invite individual members by email address from your organization's Members tab.
+## Member roles (available by request)
 
-You can also create an **invite link** from your organization's Settings tab. Any number of individuals can use the same invite link.
+> **Member roles are a new feature that is currently available by request.** Please contact support@apollographql.com to enable member roles for your organization.
+>
+> If you're using member roles, [please let us know what you think!](https://forms.gle/kdXBeD7jJXTYMQna7) Your feedback will help us improve the feature before we roll it out to all organizations.
+>
+> If roles are **not** enabled for your organization, all members can perform all actions described in [Role permissions](#role-permissions).
 
-> **Do not share your invite link publicly.** Anyone with the link can join your organization. If your invite link becomes compromised, you can replace or disable it from the Settings tab.
+Each organization member has one of the following roles:
 
-## Removing members
+| Name | Description |
+|------|-------------|
+| `Admin` | Has management access to the entire organization, including its members, graphs, and configuration. |
+| `Contributor` | Has management access to the organization's graphs, but not to members or configuration. Intended for back-end developers who make changes to graphs. |
+| `Observer` | Has view-only access to the organization's graph data, such as schemas and metrics. Intended for analysts interested in back-end performance data. |
+| `Consumer` | Has view-only access to the organization's graph _schemas_, but not metrics. Intended for developers building client applications against a graph. |
+| `Billing Manager` | Has management access to the organization's configuration and billing. Can also remove members (but not invite them). No access to graphs.  |
+
+Only admins can assign roles to other members. You can see which role you have in a particular organization from your [user settings page](https://studio.apollographql.com/user-settings), or from that organization's Members tab.
+
+### Role permissions
+
+<table>
+<thead>
+<tr>
+<th>Action</th>
+<th>Admin</th>
+<th>Billing Manager</th>
+<th>Contributor</th>
+<th>Observer</th>
+<th>Consumer</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>Invite members</td><td align="center">✓</td><td></td><td></td><td></td><td></td>
+</tr>
+<tr>
+<td>Remove members</td><td align="center">✓</td><td align="center">✓</td><td></td><td></td><td></td>
+</tr>
+<tr>
+<td>View and edit billing information</td><td align="center">✓</td><td align="center">✓</td><td></td><td></td><td></td>
+</tr>
+<tr>
+<td>Manage organization configuration (name, avatar, etc.)</td><td align="center">✓</td><td align="center">✓</td><td></td><td></td><td></td>
+</tr>
+<tr>
+<td>Create, rename, and delete graphs</td><td align="center">✓</td><td></td><td align="center">✓</td><td></td><td></td>
```

Comment:
Just reiterating that this is inaccurate right now (contributors can't delete graphs), but I know this is under discussion. @daniman @ndintenfass is there a final decision here?

Reply:
Unchanged for now –– only admins can delete graphs. We updated the table accordingly.